src/tree/SiteTreeItem.ts

```diff
@@ -51,62 +48,52 @@ export abstract class SiteTreeItem extends SiteTreeItemBase implements ISiteTree
         }
     }
 
-    public get client(): SiteClient {
-        return this.root.client;
-    }
-
     public get defaultHostUrl(): string {
-        return this.root.client.defaultHostUrl;
+        return this.site.defaultHostUrl;
     }
 
     public get defaultHostName(): string {
-        return this.root.client.defaultHostName;
-    }
-
-    public async openInPortal(): Promise<void> {
```

Comment:
why is openInPortal() removed?

Reply:
This is an old pattern and it's not used anymore. The other place that used to do it was `AzureTreeItem`, which was deleted during the ui package correlation id PR: https://github.com/microsoft/vscode-azuretools/pull/963/files#diff-e2ec57a2fe686027c0075dfb05b4b485c69b829e08ced1aa205cf6135dcef18cL487